React/resume-generator/src/components/InputComponent.tsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react"
+import FieldProps from "../interfaces/FieldProps"
+import styles from "./InputComponent.module.css"
+
+
+function InputField({label, placeholder, value, id, size}: FieldProps) {
+
+    /** state for if user is editing the field */
+    const [isUserEditing, setIsUserEditing] = useState(false)
+    /** state for currently typed selection into input box */
+    const [inputValue, setInputValue] = useState(placeholder)
+
+
+    function onChange(event: React.FormEvent<HTMLInputElement>) {
+        setInputValue(event.currentTarget.value)
+    }
+
+    /** Returns edit button when static text displayed*/
+    function editButton() {
+        return (
+            <button className="edit-btn-small" onClick={() => setIsUserEditing(true)}>Edit</button>
+        )
+    }
+
+    /** Returns accept button when input box displayed */
+    function acceptButton() {
+        return (
+            <button className="accept-btn-small" onClick={() => setIsUserEditing(false)}>Accept</button>
+        )
+    }
+
+    /** Returns input box when user is editing text */
+    function inputField({ placeholder, value, id, size}: FieldProps) {
+        if (size == "small") {
+            var btnStyle = styles.inputSmall
```

Comment:
Don't use var. Before the if do "let btnStyle = styles.inputLarge" then if (size === "small") { btnStyle = styels.inputSmall; }